bpf/bpf_overlay.c

```diff
@@ -72,6 +72,13 @@ static inline int handle_ipv6(struct __sk_buff *skb, __u32 *identity)
 		if (unlikely(skb_get_tunnel_key(skb, &key, sizeof(key), 0) < 0))
 			return DROP_NO_TUNNEL_KEY;
 		*identity = key.tunnel_id;
+
+		/* Any node encapsulating will map any HOST_ID source to be
+		 * presented as REMOTE_NODE_ID, therefore any attempt to signal
+		 * HOST_ID as source from a remote node can be droppped. */
```

Comment:
I think we need this treatment for the `ipv6_store_flowlabel()` call in `bpf_lxc.c` as well?

Reply:
I had looked at this and the only use of `ipv6_store_flowlabel()` is this:
```
        if (ipv6_store_flowlabel(skb, l3_off, SECLABEL_NB) < 0)
```

`SECLABEL_NB` is always guaranteed to be an endpoint identity. That said, adding a check for HOST_ID and dropping such attempts seems correct so I will add that.

Reply:
@joestringer I've added a commit to drop such packets.